ice_setup/ice.py

```diff
@@ -1454,11 +1500,9 @@ def _main(argv=None):
     parser.catches_help()
     parser.catches_version()
 
-    # when no arguments are passed in, just use our default routine
-    if not parser.arguments:
-        sudo_check()
-        default()
-
+    # when no subcommands are passed in, just use our default routine
```

Comment:
I don't follow you.  I changed the comment just for that reason, I said when "no subcommands", which on this version the only sumbcommand is "configure".  I also removed the check on parser.arguments.  So I think what I have is correct.